pkgs/applications/science/logic/spass/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
 
   sourceRoot = ".";
 
-  nativeBuildInputs = [ bison flex ];
+  nativeBuildInputs = [ bison flex gcc ];
```

Comment:
This almost never works, using gccStdenv is the correct way to build a package with gcc on darwin.